apps/src/code-studio/pd/workshop_dashboard/components/workshop_form.jsx

```diff
@@ -505,7 +505,11 @@ export class WorkshopForm extends React.Component {
   renderSubjectSelect(validation) {
     if (this.shouldRenderSubject()) {
       const options = Subjects[this.state.course].map((subject, i) => {
-        return (<option key={i} value={subject}>{subject}</option>);
+        // Temporarily hiding CSF 201 subject from workshop dashboard UI
+        // until the pilot is over in winter 2018
+        if (subject !== "Deep Dive Workshop") {
+          return (<option key={i} value={subject}>{subject}</option>);
+        }
```

Comment:
This will evaluate to `null` for "Deep Dive Workshop", which probably _works_ but is kind of dirty. I'd prefer to filter it out properly, something like this:

```jsx
const included_subjects = Subjects[this.state.course]
  .filter(subject => subject !== "Deep Dive Workshop");

const options = included_subjects.map(subject => (
  <option key={i} value={subject}>{subject}</option>
 ));
```

Reply:
Also let's add a `TODO: ` comment with an owner to remove this once the pilot is over.